[2023-03-13 19:39:49,009][DEBUG]: Starting new HTTP connection (1): localhost:8080
[2023-03-13 19:39:49,015][DEBUG]: http://localhost:8080 "GET / HTTP/1.1" 404 None
[2023-03-13 19:39:49,017][DEBUG]: Reproduce HTTP request with:
[2023-03-13 19:39:49,017][DEBUG]: curl -X POST 'http://localhost:8080/evaluateFromRepoWithParams' --data 'programName=../../../../disk/results/chasebench/stb-128/vadalog-parsimonious-naive/data/q11/run-0/new_program.vada' --data 'params=NonExist=1' --data 'prop=NonExistingProp=0'
[2023-03-13 19:39:49,018][DEBUG]: Starting new HTTP connection (1): localhost:8080
[2023-03-13 19:49:48,740][ERROR]: Interrupted!
